torchtext/utils.py

```diff
@@ -78,9 +78,12 @@ def _process_response(r, root, filename):
         root, filename = os.path.split(path)
 
     if not os.path.exists(root):
-        raise RuntimeError(
-            "Download directory {} does not exist. "
-            "Did you create it?".format(root))
+        if root == ".data":
```

Comment:
should we use `try`/`except`? And `root` could be a user-defined directory.

Reply:
Sure. Update in 091e263